src/main/java/uk/gov/companieshouse/reconciliation/company/CompanyCountTrigger.java

```diff
@@ -19,6 +26,10 @@ public void configure() throws Exception {
                 .setHeader("TargetName", simple("MongoDB"))
                 .setHeader("Comparison", simple("company profiles"))
                 .setHeader("Destination", simple("{{endpoint.output}}"))
+                .setHeader("Upload", simple("{{endpoint.s3.upload}}"))
+                .setHeader("Presign", simple("{{endpoint.s3presigner.download}}"))
+                .setHeader(AWS2S3Constants.KEY, simple("company/count_${date:now:yyyyMMdd}.csv"))
```

Comment:
Would also include the time of the comparison (e.g. to avoid overwriting existing files if we produce multiple comparisons on the same day at some point).